PWGHF/Tasks/CMakeLists.txt

```diff
@@ -9,6 +9,11 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
+o2physics_add_dpl_workflow(correlate-estimator-dn-deta
+                    SOURCES correlateEstimatorDnDeta.cxx
```

Comment:
Please start the name with `task`, e.g. `taskMultiplicityEstimatorCorrelation.cxx`.